src/pages/_document.tsx

```diff
@@ -7,11 +7,11 @@ export default function Document() {
       <body>
         <Main />
         <NextScript />
-        <div id="nypl-footer"></div>
+        {/* <div id="nypl-footer"></div>
```

Comment:
did we say we were going to add the footer back?

Reply:
Yes we did. This won't be merged into main, but it needs to be VQAd like this